client/my-sites/google-my-business/google-my-business-stats/index.js

```diff
@@ -0,0 +1,344 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import page from 'page';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import SectionHeader from 'components/section-header';
+import Button from 'components/button';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import Main from 'components/main';
+import Card from 'components/card';
+import StatsNavigation from 'blocks/stats-navigation';
+import { recordTracksEvent } from 'state/analytics/actions';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormSelect from 'components/forms/form-select';
+import { createNotice } from 'state/notices/actions';
+import NotEnoughData from './not-enough-data';
+
+class Stats extends Component {
+	static propTypes = {
+		recordTracksEvent: PropTypes.func.isRequired,
+		translate: PropTypes.func.isRequired,
+	};
+
+	componentDidMount() {
+		this.props.createNotice( 'is-success', 'Connected to Google My Business' );
+	}
+
+	goBack = () => {
+		page.back( `/google-my-business/${ this.props.siteId }` );
+	};
+
+	render() {
+		const { translate } = this.props;
+		const notEnoughData = true;
+		return (
+			<Main wideLayout>
+				<SidebarNavigation title="Stats" />
+				<div className="google-my-business-stats google-my-business">
+					<StatsNavigation selectedItem={ 'google-my-business' } />
+
+					<Card className="google-my-business-stats__details">
+						<div className="google-my-business-stats__container">
+							<img
+								src="/calypso/images/google-my-business/cookies.png"
+								className="google-my-business-stats__image"
+								alt="gravatar"
+							/>
+
+							<div className="google-my-business-stats__text">
+								<div className="google-my-business-stats__header">Cate's Cookies</div>
+								<div className="google-my-business-stats__address">
+									<div className="google-my-business-stats__listing-detail">
+										345 North Avenue<br />Talihassee, FL 34342<br />USA
+									</div>
+								</div>
+								<div className="google-my-business-stats__verified">
+									<Gridicon icon="checkmark-circle" size={ 18 } /> Verified
+								</div>
+							</div>
+						</div>
+
+						<div className="google-my-business-stats__listing-button-wrapper">
+							<Button
+								className="google-my-business-stats__listing-button"
+								href="https://business.google.com"
+								target="_blank"
+							>
+								Update Listing
+								<Gridicon icon="external" />
+							</Button>
+						</div>
+					</Card>
+
+					<SectionHeader label={ translate( 'How customers search for your business' ) } />
+					<Card>
+						<FormFieldset>
+							<FormSelect className="google-my-business-stats__select">
+								<option>{ translate( '1 week' ) }</option>
```

Comment:
Hi! I've found a possible matching string that has already been translated **58** times:
`translate( 'Week' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/gmb-prototype)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).